src/ompl/CMakeLists.txt

```diff
@@ -36,15 +36,20 @@ source_group("OMPL Headers" FILES "${OMPL_HEADERS}")
 # build the library
 if(MSVC)
     add_library(ompl STATIC ${OMPL_SOURCE_CODE})
-else(MSVC)
+else()
     add_library(ompl SHARED ${OMPL_SOURCE_CODE})
-endif(MSVC)
+endif()
 target_link_libraries(ompl
     ${OMPL_LINK_LIBRARIES}
     ${Boost_SERIALIZATION_LIBRARY}
     ${Boost_FILESYSTEM_LIBRARY}
     ${Boost_SYSTEM_LIBRARY}
-    ${CMAKE_THREAD_LIBS_INIT})
+    ${CMAKE_THREAD_LIBS_INIT}
+    Boost::filesystem
```

Comment:
Linking alias targets ensures they actually exist, vs variables which can silently fail.

Reply:
Don't you need to add Boost::serialization and delete ${Boost_SERIALIZATION_LIBRARY} and ${Boost_FILESYSTEM_LIBRARY}?

Reply:
`Boost::serialization` is already there, but yes, I forgot to remove the old properties. Great catch. 
While I'm at it, I'll sort them alphabetically. 

Fixed in a9c2ba50184365cec6599e011346fa6b30ce329f